common/changes/xingwa-a11y_2017-01-10-23-54.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Dropdown: screen reader for first focus option. ",
```

Comment:
The comment isn't clear; can you fix the grammar here? I don't completely understand the change.

Adding @eliblock12